backend/Users/Users.ts

```diff
@@ -71,3 +65,16 @@ export async function findUser(email, password) {
     return null;
   }
 }
+
+export async function findUserByEmail(email) {
```

Comment:
Particularly because this is a reusable function, it's nice to type the function signature so that every part of the code that uses it gets good autocompletion:

```suggestion
export async function findUserByEmail(email: string): Promise<User | null> {
```

Reply:
Complete.